ionic2/app/components/map/map.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Directive, Input, ElementRef, OnInit } from '@angular/core';
+
+//let PokeMap = require('pokemap-1');
+//let PokeMap = require('pokemap-2');
+
+// Dummy PokeMap class until PokeMaps are implemented
+let PokeMap: any = function(...args) {console.debug('map:constructor', ...args)};
```

Comment:
I'd ES6 Arrow functions for one-liners here. Looks a bit cleaner. Just a detail I noticed